scripts/generate_all_jobs.py

```diff
@@ -58,6 +58,13 @@ def main(argv=sys.argv[1:]):
     print('')
 
     config = get_index(args.config_url)
+    if args.config_url.startswith("file:"):
+        print('!WARNING!')
+        print('! Local file system path used for configuration.')
+        print('! Configuration will not be accessible to jobs during execution.')
+        print('! Consider using a web(http) hosted configuration repository.')
+        print('')
```

Comment:
Please print warnings to `stderr`. Also a shorter message would be nice. I don't think this should be five lines long and contain `!` around each line.

Reply:
Thanks for the comment. Updated.